Gruntfile.js

```diff
@@ -65,11 +75,13 @@ module.exports = function(grunt) {
             ignoreLeaks: false,
             grep: 'food'
           },
+          growlOnFail: false,
+          growlOnSuccess: false,
 
           reporter: 'Nyan',
 
           // URLs passed through as options
-          urls: ['http://localhost:' + port + '/example/test/test2.html'],
+          urls: ['http://localhost:' + port + '/example/test/test2.html']
```

Comment:
Generally prefer the trailing comma since it leads to fewer lines changed if you add a new item to the parent dict.

Reply:
Changed again